go/fasthttp/go.mod

```diff
@@ -0,0 +1,5 @@
+module main
+
+require github.com/valyala/fasthttp v1.5.0
+
+go 1.13
```

Comment:
this line is not required

Reply:
It's automatically added by Go 1.13 each time you run any of the `go mod` commands.